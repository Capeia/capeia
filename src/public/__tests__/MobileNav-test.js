// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<MobileNav>', () => {
  it('renders correctly', () => {
    jest.deepUnmock('../MobileNav')
    const Component = require('../MobileNav').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
