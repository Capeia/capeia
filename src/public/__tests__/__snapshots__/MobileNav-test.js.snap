// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MobileNav> renders correctly 1`] = `
<nav
  className="MobileNav"
  role="navigation"
>
  <h1>
    Ressort
  </h1>
  <ul>
    <li>
      <a
        activeClassName="active"
        href="/stem-cells"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/stem_cells.svg"
        />
        Stem Cells
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/brain-and-nerves"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/brain_nerves.svg"
        />
        Brain & Nerves
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/cancer"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/cancer.svg"
        />
        Cancer
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/zoology"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/zoology.svg"
        />
        Zoology
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/paleobiology"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/paleontology.svg"
        />
        Paleobiology
      </a>
    </li>
    <li>
      <a
        activeClassName="active"
        href="/planetary-science"
        onClick={[Function]}
      >
        <img
          alt=""
          src="sections/planetary_science.svg"
        />
        Planetary Science
      </a>
    </li>
  </ul>
</nav>
`;
